.pnpmfile.cjs

```diff
@@ -69,6 +69,7 @@ function fixDeps( pkg ) {
 		] ) {
 			pkg.optionalDependencies[ dep ] = '*';
 		}
+		pkg.optionalDependencies[ 'change-case' ] = '^4';
```

Comment:
Probably could have removed it from line 50 too, but no harm done other than a few lost microseconds. On every single run. 😆

Reply:
I may follow this up with something that sets deps for everything instead of doing `'*'` and hoping it works. But first I wanted to get Simple deploys unblocked.